certbot-apache/setup.py

```diff
@@ -10,14 +12,19 @@
 # acme/certbot version.
 install_requires = [
     'acme>=0.29.0',
-    'certbot>=1.1.0',
-    'mock',
+    'certbot>=1.4.0.dev0',
```

Comment:
Is the only reason for this change `pip`'s issues with dependency resolution? I think it's totally fine to make this change, but I was initially thinking we wouldn't have to do this.

Reply:
I forgot to revert this change, I was just testing things. Nice catch!